modules/4-graphql.livemd

```diff
@@ -88,58 +125,51 @@ OWASP recommends explicitly defining and enforcing all API response payload sche
 
 ```elixir
-# -------------------------------------------------------------
-# answer = 1
-#
-# HTTP/2 401 Unauthorized
-# Date: Tues, 16 Aug 2022 21:06:42 GMT
-# …
-# {
-# 	“error”:”token expired”
-# {
-# -------------------------------------------------------------
-# answer = 2
-#
-# HTTP/2 200 OK
-# Date: Tues, 16 Aug 2021 22:06:42 GMT
-# …
-# {
-# 	“errors”:[
-# 		{
-# 			“locations”:[
-# 			{
-# 				“column”:2,
-# 				:line”:2
-# 			}
-# 			],
-# 			“message”: “Parsing failed at
-# 		}
-# 	]
-# }
-# --------------------------------------------------------------
-# answer = 3
-#
-# HTTP/2 200 OK
-# Date: Tues, 16 Aug 2022 21:06:42 GMT
-# …
-# {
-# 	“error”:”ID token for user 55e4cb07 at org 1234 expired”
-# {
-# ---------------------------------------------------------------
-# answer = 4
-#
-# HTTP/2 404 File Not Found
-# Date: Tues, 16 Aug 2022 21:06:42 GMT
-# …
-# {
-# 	“error”:”/www/home/file.txt not found ”
-# {
-# ---------------------------------------------------------------
-
-IO.puts(answer)
+result =
+  (
+    input =
+      Kino.Input.select(
+        "Choose your answer",
+        [none: "", a: "Option 1", b: "Option 2", c: "Option 3", d: "Option 4"],
+        default: :none
+      )
+
+    Kino.render(input)
+    Kino.Input.read(input)
+  )
+
+[module_id, question_id] =
+  "# GRAPHQL:2\n\n# -------------------------------------------------------------\n# Option 1\n#\n# HTTP/2 401 Unauthorized\n# Date: Tues, 16 Aug 2022 21:06:42 GMT\n# …\n# {\n# \t“error”:”token expired”\n# {\n# -------------------------------------------------------------\n# Option 2\n#\n# HTTP/2 200 OK\n# Date: Tues, 16 Aug 2021 22:06:42 GMT\n# …\n# {\n# \t“errors”:[\n# \t\t{\n# \t\t\t“locations”:[\n# \t\t\t{\n# \t\t\t\t“column”:2,\n# \t\t\t\t:line”:2\n# \t\t\t}\n# \t\t\t],\n# \t\t\t“message”: “Parsing failed at\n# \t\t}\n# \t]\n# }\n# --------------------------------------------------------------\n# Option 3\n#\n# HTTP/2 200 OK\n# Date: Tues, 16 Aug 2022 21:06:42 GMT\n# …\n# {\n# \t“error”:”ID token for user 55e4cb07 at org 1234 expired”\n# {\n# ---------------------------------------------------------------\n# Option 4\n#\n# HTTP/2 404 File Not Found\n# Date: Tues, 16 Aug 2022 21:06:42 GMT\n# …\n# {\n# \t“error”:”/www/home/file.txt not found ”\n# {\n# ---------------------------------------------------------------\n\ninput = Kino.Input.select(\"Choose your answer\", [\n  none: \"\",\n  a: \"Option 1\", \n  b: \"Option 2\", \n  c: \"Option 3\",\n  d: \"Option 4\"\n], default: :none)\n\nKino.render(input)\n\nKino.Input.read(input)"
```

Comment:
Is there a reason you're doing it as one line and not a heredoc?

Reply:
This style will be harder to edit

Reply:
This is just how the smart cell saved the code. There might be a way to force the heredoc though

Reply:
i see

Reply:
I'll see if there's a way to clean this raw code up. 

I was only editing through the smart cell itself, so I didn't realize how unreadable the rendered code is